src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -3974,6 +3984,27 @@ type Trip implements Node {
     ): [Alert]
 }
 
+"""
+Trip on a specific date
+"""
+type DatedTrip implements Node {
+    id: ID!
+    trip: Trip!
+    date: String!
```

Comment:
I think we want to use a scalar for this. @optionsome is already working on that in his trip connection PR: https://github.com/opentripplanner/OpenTripPlanner/pull/5185

Reply:
Done in https://github.com/opentripplanner/OpenTripPlanner/pull/5393/commits/20df02b87e3598524100587dc9fbe30ee87117e8